.github/workflows/swift-toolchain.yml

```diff
@@ -3669,10 +3741,10 @@ jobs:
               -p:INCLUDE_AMD64_SDK=true `
               -p:INCLUDE_X86_SDK=true `
               -p:INCLUDE_ARM64_SDK=true `
-              -p:ANDROID_INCLUDE_ARM64_SDK=true `
-              -p:ANDROID_INCLUDE_x86_64_SDK=true `
-              -p:ANDROID_INCLUDE_ARM_SDK=true `
-              -p:ANDROID_INCLUDE_X86_SDK=true `
+              -p:ANDROID_INCLUDE_ARM64_SDK=${{ inputs.build_arch == 'amd64' && true || false }} `
+              -p:ANDROID_INCLUDE_x86_64_SDK=${{ inputs.build_arch == 'amd64' && true || false }} `
+              -p:ANDROID_INCLUDE_ARM_SDK=${{ inputs.build_arch == 'amd64' && true || false }} `
+              -p:ANDROID_INCLUDE_X86_SDK=${{ inputs.build_arch == 'amd64' && true || false }} `
```

Comment:
Perhaps we should just have a `build_android` setting that is global and use that throughout?

Reply:
How do we use a global or workflow-level env variable that depends on `inputs` (like `inputs.build_arch`) here? @compnerd

Reply:
You can define a new parameter for the workflow for that. @Steelskin setup a large number of these. I think for the `inputs` case, we can add a new boolean option.

Reply:
I believe you should be able to add a `build_android` boolean option to the `swift-toolchain` workflow.
You should be able to infer its value when you call the workflow from `build-toolchain`. It should only be set to true for the Windows sub-workflow, if `needs.context.outputs.windows_build_arch` is `amd64`. Set it to false for the Mac sub-workflow.

Reply:
Done